pkg/host/configvalidation.go

```diff
@@ -267,10 +267,21 @@ func (v *configValidationImpl) RuntimeConfigApplied(device *v1alpha1.NicDevice)
 		}
 	}
 
+	qosConfigurationRequested := desiredTrust != "" || desiredPfc != ""
+
+	if !qosConfigurationRequested {
```

Comment:
I don't know if this makes it even more complicated understanding the code.
Isn't it easier to understand to say "if both are empty" instead of negating things?
```suggestion
	// Return early if no QoS configuration is requested.
	if desiredTrust == "" && desiredPfc == "" {
```

Reply:
The user might provide custom values for trust and pfc and leave one empty